python/lsst/pipe/tasks/mergeDetections.py

```diff
@@ -347,10 +412,18 @@ def cullPeaks(self, catalog):
             familySize = len(oldPeaks)
             totalPeaks += familySize
             for rank, peak in enumerate(oldPeaks):
-                if ((rank < self.config.cullPeaks.rankSufficient)
-                    or (sum([peak.get(k) for k in keys]) >= self.config.cullPeaks.nBandsSufficient)
-                    or (rank < self.config.cullPeaks.rankConsidered
-                        and rank < self.config.cullPeaks.rankNormalizedConsidered * familySize)):
+                if (
+                    (rank < self.config.cullPeaks.rankSufficient)
+                    or (
+                        sum([peak.get(k) for k in keys])
+                        >= self.config.cullPeaks.nBandsSufficient
+                    )
+                    or (
+                        rank < self.config.cullPeaks.rankConsidered
+                        and rank
+                        < self.config.cullPeaks.rankNormalizedConsidered * familySize
+                    )
+                ):
                     keptPeaks.append(peak)
                 else:
                     culledPeaks += 1
```

Comment:
Most of the changes here appear to be formatting changes. Is it possible to separate out the code changes and the formatting changes.